app/assets/javascripts/kuroko2/job_instances.js

```diff
@@ -3,11 +3,28 @@
 
 jQuery(function ($) {
   var logIntervalId;
+  var notifyIfNeeded = function (status, name) {
+    if (!('Notification' in window)) {
+      return;
+    }
+
+    if (Notification.permission === 'granted' && Cookies.get('notification') === 'on') {
+      var notification = new Notification(
+        "[" + status + "] " + name,
+        {"icon": window.location.origin + "/assets/kuroko2/kuroko-logo-" + status.toLowerCase() + ".png"}
```

Comment:
Oops... I'm sorry, what I pointed out is mistake 😢 
But the same situation will occur using `config.assets.prefix` or` config.action_controller.asset_host`, right?